test/CMakeLists.txt

```diff
@@ -1342,27 +1342,28 @@ add_custom_test(test_conv_igemm_dynamic_xdlops_bwd_float SKIP_UNLESS_ALL HALF_DI
 COMMAND ${DYNAMIC_IMPLICITGEMM_BWD_ENVS_XDLOPS} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --verbose --input  4  512 128 128 --weights 12  512  1 1 --pads_strides_dilations 0 0 1 1 1 1 --disable-forward --disable-backward-weights
 )
 
-#add_custom_test(test_conv_igemm_dynamic_xdlops_fwd SKIP_UNLESS_ALL HALF_ENABLED GFX90A_DISABLED GFX900_DISABLED GFX906_DISABLED SKIP_XNACK_ON
+# Commented out due to WORKAROUND_SWDEV_306318
```

Comment:
AFAICS `C % 8 == 0` in all these tests, so this W/A in tests is not needed at all, right?

Reply:
I mean we can enable all test cases.

Reply:
@atamazov  I see, thank you.  Updated